### PR TITLE
Push Host Reaper metrics

### PR DESCRIPTION
**Pull request by Glutexo**
Use a Prometheus Pushgateway to report metrics from the [Host Reaper](https://github.com/Glutexo/insights-host-inventory/blob/2fc6fd481bd3841d6fa40149f231146a0ae76192/host_reaper.py). A new [configuration value / environment variable](https://github.com/Glutexo/insights-host-inventory/blob/2fc6fd481bd3841d6fa40149f231146a0ae76192/app/config.py#L46) is used for the target.

### CODE REVIEW

File: host_reaper.py [hunk 1]
```diff
@@ -30,33 +39,45 @@ def _init_db(config):
...
+    init_tasks(config)
+
+    registry = CollectorRegistry()
+    for metric in COLLECTED_METRICS:
+        registry.register(metric)
+
+    Session = _init_db(config)
+    session = Session()
+
+    run(config, session)
```
Comment (jharting) [inline host_reaper.py]: Can we add a metric for when this fails?

File: host_reaper.py [hunk 2]
```diff
@@ -16,11 +21,15 @@
 from tasks import init_tasks
 
 
-__all__ = ("main",)
+__all__ = ("main", "run")
+
+PROMETHEUS_JOB = "host_reaper"
```
Comment (jharting) [inline host_reaper.py]: Make sure to update this to reflect the kubernetes namespace (`NAMESPACE` env var)

File: host_reaper.py (hunk 1 above)
Comment (Glutexo) [reply host_reaper.py]: Sure.

**Comment by Glutexo** [comment]
Rebased. I see some failed tests. Gonna fix.

**Comment by Glutexo** [comment]
Tests are fixed. The PR should be now mergeable.

**Pull request merged**